lib/santa_maria/gateway/santa_maria_v2.rb

```diff
@@ -85,6 +92,13 @@ def new_variant(sku)
         variant.ready_mix = !sku['tintedOrReadyMix'].eql?('Tinted')
         variant
       end
+
+      def new_color(color_data)
+        color = SantaMaria::Domain::Color.new
```

Comment:
for future reference you can also this to save having to asign/return at the end:

```
SandaMaria::Domain::Color.new.tap do |color|
  color.color_id = color_data['colorID']
  color.rgb = color_data['rgb']
end
```

Reply:
Nice.